src/Symfony/Component/ExpressionLanguage/Tests/ExpressionLanguageTest.php

```diff
@@ -287,4 +287,11 @@ function (ExpressionLanguage $el) {
             ],
         ];
     }
+
+    public function testParseAlreadyParsedExceptionReturnsSameObject()
```

Comment:
Exception?

Reply:
I meant to say "Expression" actually... thanks!